frontend/package.json

```diff
@@ -22,8 +22,8 @@
     "translation:check": "node ./tasks/checkTranslation.js"
   },
   "engines": {
-    "node": ">=12.11",
-    "npm": ">=6.13.4"
+    "node": ">=16.13.0",
```

Comment:
I'd restrict node version by latest 17 that is supported. Than there should be at least warning with unsupported node